### DISCUSSION TITLE
A function of the Tag Class take minutes to run

### DISCUSSION THREAD

**Discussion by Hydrog3n** [PrestaShop/PrestaShop#34401] [Q&A]
Hello Prestashop community,

We notice when we upload product imports csv, that a call to the method `updateTagCount` is done here: https://github.com/PrestaShop/PrestaShop/blob/develop/controllers/admin/AdminImportController.php#L1459

It took more than 1 minute, we dig into it and discovered the SQL request depending on the number of products (we have 5k) and the number of user groups (we have 60).  

So for the moment, we comment the call but we want to know if others get into the same trouble. 

We also see the tag_count isn't really used in the project, maybe we miss it so what is the usage of it? 

Thank you for the help

**Comment by MatShir** [comment]
Maybe something that can be removed, could make a exhaustive where the tag count is used ?

**Reply by Hydrog3n** [reply]
Hello, I didn't found anywhere maybe in the search index but I don't have good knowledge of that part
